Source/Events.Processing/IEventProcessingRequestConverter.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using PbCommittedEvent = Dolittle.Runtime.Events.Contracts.CommittedEvent;
+using PbStreamEvent = Dolittle.Runtime.Events.Processing.Contracts.StreamEvent;
+
+namespace Dolittle.SDK.Events.Processing
+{
+    /// <summary>
+    /// Defines a system that can convert a request for processing an event to <see cref="EventContext" /> and the CLR instance of the event.
+    /// </summary>
+    public interface IEventProcessingRequestConverter
+    {
+        /// <summary>
+        /// Gets the <see cref="EventContext" /> from the <see cref="PbCommittedEvent" />.
+        /// </summary>
+        /// <param name="event">The <see cref="PbCommittedEvent" />.</param>
+        /// <returns>The derived <see cref="EventContext" />.</returns>
+        EventContext GetEventContext(PbCommittedEvent @event);
```

Comment:
```suggestion
        /// <summary>
        /// Converts a <see cref="PbCommittedEvent" /> to an <see cref="EventContext" />.
        /// </summary>
        /// <param name="event">The <see cref="PbCommittedEvent" />.</param>
        /// <returns>The derived <see cref="EventContext" />.</returns>
        EventContext ToEventContext(PbCommittedEvent @event);
```

As this is a converter shouldn't we be using the `ToSomething` wording here instead of `Get`? Kinda like how we had it with the extension methods? On the extension methods the `ToSomething` worked a bit nicer as it was extending the type but I think it still works here.

For maximum verbosity we could go for `ConvertProtobufEventToEventContext` or `ConvertToEventContext` or something along those lines (which I like neither off).

Reply:
Maybe "Converter" is the wrong name here. It does not really convert the message, it gets certain parts of it, for example the event context or the event itself. And given that context i don't think that `ToEventContext` makes sense